docs/integrations/webhooks/sync-data.mdx

```diff
@@ -353,13 +353,13 @@ This guide can be adapted to listen for any Clerk event.
 
   In the following example, the `if` statement will narrow the type to the `user.created` type and using `evt.data` will give you autocompletion and type safety for that event.
 
-  ```diff {{ prettier: false, filename: 'app/api/webhooks/route.ts' }}
-  - console.log(`Webhook with and ID of ${id} and type of ${eventType}`)
-  - console.log('Webhook body:', body)
+  ```jsx {{ prettier: false, filename: 'app/api/webhooks/route.ts', del: [1, 2], ins: [[4, 6]] }}
```

Comment:
Despite the `filename` this snippet is valid JavaScript so formatting would still work. Having said that, this is a great callout because the filename is `.ts` so we should totally make the language `ts`. We may want to show a little TS icon for `ts` code blocks in the future, or something like that.

I've also replaced all of the `jsx` blocks in this PR with `js` unless JSX is actually used, because we may want to differentiate between the two in future (e.g. show a little React icon for `jsx` blocks)